### DISCUSSION TITLE
Incompatibility in v3.5.14; causing error with wire:sortable

### DISCUSSION THREAD

**Discussion by eneadrian** [livewire/livewire#9046] [7. Bugs (archived)] [answered]
After upgrading from version 3.5.12 to 3.5.14, a previously functional implementation of wire:sortable started causing an error. This issue only occurs in the newer version (3.5.14), and downgrading to 3.5.12 resolves the problem.

The error appears to be related to the wire:sortable functionality or its options and how it interacts with dynamic Livewire components.

**Expected Behavior**
The code should function identically to how it works in version 3.5.12, where the wire:sortable directive and its options are correctly applied without raising errors.

```
<div wire:sortable="updatePosition" wire:sortable.options="{ animation: 700 }" class="space-y-6">
	@foreach($sections as $section)
		<livewire:dynamic-component
			component="app.sections.components.{{ $section->parent->slug }}"
			:key="$section->id" />
	@endforeach
</div>
```

**Actual Behavior**
An error is thrown when using the same code in version 3.5.14. The exact error details are included in the attached screenshots.

PS: No changes were made to the Blade component or underlying data between testing 3.5.12 and 3.5.14.

**Comment by eneadrian** [comment, marked as answer]
this was fixed in last commit (e94a2cd) from @calebporzio **thanks**!

**Reply by d2inco** [reply]
Was this definitively fixed?

I am experiencing this in 3.5.18.  I stumbled on issue #9046 and #9047, indicating that it was fixed.... but unsure if it got broken again...
